src/ResourceManager/Resources/Commands.Resources/help/Save-AzureRmDeploymentTemplate.md

```diff
@@ -0,0 +1,197 @@
+---
+external help file: Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml
+Module Name: AzureRM.Resources
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.resources/save-azurermdeploymenttemplate
+schema: 2.0.0
+---
+
+# Save-AzureRmDeploymentTemplate
+
+## SYNOPSIS
+Saves a deployment template to a file.
+
+## SYNTAX
+
+### SaveByDeploymentName (Default)
+```
+Save-AzureRmDeploymentTemplate -DeploymentName <String> [-Path <String>] [-Force] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### SaveByDeploymentObject
+```
+Save-AzureRmDeploymentTemplate -DeploymentObject <PSDeployment> [-Path <String>] [-Force]
+ [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Save-AzureRmDeploymentTemplate**  cmdlet saves a deployment template to a JSON file.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Save-AzureRmDeploymentTemplate -DeploymentName "TestDeployment"
+```
+
+This command gets the deployment template from TestDeployment and saves it as a JSON file in the current directory.
+
+### Example 2: Get a deployment and save its template
+```
+PS C:\>Get-AzureRmDeployment -Name "RolesDeployment" | Save-AzureRmDeploymentTemplate
+```
+
+This command gets the deployment "RolesDeployment" at the current subscription scope and saves its template.
+
+## PARAMETERS
+
+### -ApiVersion
+When set, indicates the version of the resource provider API to use.
+If not specified, the API version is automatically determined as the latest available.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DeploymentName
+The deployment name.
+
+```yaml
+Type: String
+Parameter Sets: SaveByDeploymentName
+Aliases: Name
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DeploymentObject
+The deployment object.```yaml
```

Comment:
@Tiano2017 same comment about the trailing ` ```yaml ` string

Reply:
fixed.